### PR TITLE
fix(goctl model file): change model template file type

### PR DESCRIPTION
**Pull request by jsonMark**
All model template variables are stored in tpl format files with the
same name as the template generated using the template init command.

The ultimate purpose is to facilitate the customization of goctl users to upgrade the model template

### CODE REVIEW

File: tools/goctl/model/sql/template/template.go [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+//go:embed tpl/table-name.tpl
+var TableName string
+
+// New defines the template for creating model instance.
+//
+//go:embed tpl/model-new.tpl
+var New string
+
+// Tag defines a tag template text
+const Tag = "`db:\"{{.field}}\"`"
```
Comment (anqiansong) [inline tools/goctl/model/sql/template/template.go:33]: 20行为什么注释了，而 tag 模板文件还在

File: tools/goctl/model/sql/gen/gen.go [hunk 2]
```diff
@@ -309,7 +309,7 @@ func (g *defaultGenerator) genModel(in parser.Table, withCache bool) (string, er
 	var list []string
 	list = append(list, insertCodeMethod, findOneCodeMethod, ret.findOneInterfaceMethod,
 		updateCodeMethod, deleteCodeMethod)
-	typesCode, err := genTypes(table, strings.Join(modelutil.TrimStringSlice(list), pathx.NL), withCache)
+	typesCode, err := genTypes(table, strings.Join(modelutil.TrimStringSlice(list), ""), withCache)
```
Comment (anqiansong) [inline tools/goctl/model/sql/gen/gen.go:312]: 这里为啥换成空字符串

Comment (jsonMark) [reply tools/goctl/model/sql/gen/gen.go:312]: 不换成“”生成的 下面每一个方法都会有一行空格，当时是mac环境
	hwModel interface {
		Insert(ctx context.Context, data *Hw) (sql.Result, error)

		InsertTrans(session sqlx.Session, ctx context.Context, data *Hw) (sql.Result, error)

		FindOne(ctx context.Context, id int64) (*Hw, error)

		FindOneByField(ctx context.Context, data *Hw) (*Hw, error)

		Update(ctx context.Context, data *Hw) error

		Edit(ctx context.Context, data *Hw) (int64, error)

		EditTrans(session sqlx.Session, ctx context.Context, data *Hw) (int64, error)

		Delete(ctx context.Context, id int64) error
	}

我在家里window没有重现

Comment (jsonMark) [reply tools/goctl/model/sql/gen/gen.go:312]: 我今天在mac测试了，确认在【mac】下面使用typesCode, err := genTypes(table, strings.Join(modelutil.TrimStringSlice(list), pathx.NL), withCache)写法会生成
	hwModel interface {
		Insert(ctx context.Context, data *Hw) (sql.Result, error)

		FindOne(ctx context.Context, id int64) (*Hw, error)

		Update(ctx context.Context, data *Hw) error

		Delete(ctx context.Context, id int64) error
	}
这种空行

**Comment by kevwan** [comment]
closed as requested.

**Pull request closed without merging**